pkg/util/utils_test.go

```diff
@@ -57,5 +60,35 @@ func TestRetry(t *testing.T) {
 	if err := Retry(4, f); err == nil {
 		t.Fatalf("Error should have been raised by retry.")
 	}
+}
+
+func errEquals(err1 error, err2 error) bool {
+	if err1 == err2 {
+		return true
+	} else if err1 == nil || err2 == nil {
+		return false
+	} else {
+		return err1.Error() == err2.Error()
+	}
+}
+
+func TestGetLocalkubeDownloadURL(t *testing.T) {
+	inner := func(input string, expected string, expectedError error) {
+		url, err := GetLocalkubeDownloadURL(input, constants.LocalkubeLinuxFilename)
+		if !errEquals(err, expectedError) {
+			t.Fatalf("GetLocalkubeDownloadURL Expected error: %s, Actual Error: %s", expectedError, err)
```

Comment:
Cool will do, thanks.  Going forward should all tests use Errorf instead of Fatalf or is it case by case?  Thanks.